hack/verify-codecgen.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+
+cd "${KUBE_ROOT}"
+
+generated_files=`
+  find . -not \( \
+      \( \
+        -wholename './output' \
+        -o -wholename './_output' \
+        -o -wholename './release' \
+        -o -wholename './target' \
+        -o -wholename '*/third_party/*' \
+        -o -wholename '*/Godeps/*' \
+      \) -prune \
+    \) -name '*.generated.go'`
+
+# Instal codecgen necessary to generate the files.
+# TODO: This doesn't work - Travis is complaining about it:
+# pakckage github.com/ugorji/go/codec/codecgen: directory "/home/travis/gopath/src/github.com/kubernetes/kubernetes/Godeps/_workspace/src/github.com/ugorji/go/codec/codecgen" is not using a known version control system
+# !!! Error in hack/verify-codecgen.sh:41
+#   'go get -u github.com/ugorji/go/codec/codecgen' exited with status 1
+# TODO: Once fixed, enable it in Makefile.
+go get -u github.com/ugorji/go/codec/codecgen
```

Comment:
@lavalamp 
I don't know how to make it work correctly. We need to download it to have the "codecgen" tool, but I'm not sure how to make it work with Travis/Shippable. Any hints?